monai/config/type_definitions.py

```diff
@@ -67,3 +68,10 @@
 # Generic type which can represent either a numpy.ndarray or a torch.Tensor
 # Unlike Union can create a dependence between parameter(s) / return(s)
 NdarrayTensor = TypeVar("NdarrayTensor", np.ndarray, torch.Tensor)
+
+
+TensorList = Union[torch.Tensor, Sequence[torch.Tensor]]
```

Comment:
nit: seeing `TensorList` I think I would not expect it to be a Tensor itself but only a list of tensor.
However, I agree that there can be difficult to find an appropriate name.

Reply:
Maybe use `TensorOrList`?

Reply:
I updated in the latest commit.
Thanks.